root.tf

```diff
@@ -375,6 +383,9 @@ module "backend_checks_efs" {
   access_point_path            = "/backend-checks"
   policy                       = "backend_checks_access_policy"
   mount_target_security_groups = flatten([module.file_format_lambda.file_format_lambda_sg_id, module.download_files_lambda.download_files_lambda_sg_id, module.file_format_build_task.file_format_build_sg_id, module.antivirus_lambda.antivirus_lambda_sg_id, module.checksum_lambda.checksum_lambda_sg_id])
+  nat_gateway_ids              = module.shared_vpc.nat_gateway_ids
+  vpc_cidr_block               = module.shared_vpc.vpc_cidr_block
+  vpc_id                       = module.shared_vpc.vpc_id
```

Comment:
How does this all work on intg and staging when these weren't configured?

Reply:
They already existed in integration/staging so when I added the data blocks, it worked because they existed. For a new environment, they don't yet exist so it didn't work. They should never have been added as data blocks, we should always add stuff in as variables so terraform can work out the dependencies.